Description: Don't run automake with -Werror
Origin: upstream, http://pcmanfm.git.sourceforge.net/git/gitweb.cgi?p=pcmanfm/libfm;a=commit;h=35f0fe5d59c6209148660d2c3af33956fd0c6631
Bug-Debian: http://bugs.debian.org/713194
Last-Update: 2013-07-28

Index: libfm-0.1.17/configure.ac
===================================================================
--- libfm-0.1.17.orig/configure.ac	2013-07-28 23:15:05.000000000 +0200
+++ libfm-0.1.17/configure.ac	2013-07-28 23:32:13.342073577 +0200
@@ -9,6 +9,8 @@
 
 # Checks for programs.
 AC_PROG_CC
+AM_PROG_CC_C_O
+AM_PROG_AR
 AM_PROG_LIBTOOL
 
 # Checks for libraries.
Index: libfm-0.1.17/data/ui/Makefile.am
===================================================================
--- libfm-0.1.17.orig/data/ui/Makefile.am	2011-10-24 22:21:16.000000000 +0200
+++ libfm-0.1.17/data/ui/Makefile.am	2013-07-28 23:37:27.656165367 +0200
@@ -18,6 +18,6 @@
 	$(NULL)
 
 # Purge GtkBuilder UI files
-%.ui: %.glade
+.glade.ui:
 	cp $< $@
 	$(top_builddir)/src/xml-purge $@
